src/cljs/rems/application.cljs

```diff
@@ -564,13 +564,36 @@
   ;; TODO: format readonly value in user locale (give basic-field a formatted :value and :previous-value in opts)
   [basic-field opts
    [:input.form-control {:type "date"
+                         :id (id-to-name id)
                          :name (id-to-name id)
                          :class (when validation "is-invalid")
                          :defaultValue value
                          :min min
                          :max max
                          :on-change (set-field-value id)}]])
 
+(defn- option-label [value language options]
+  (let [label (->> options
+                   (filter #(= value (:key %)))
+                   first
+                   :label)]
+    (get label language value)))
+
+(defn option-field [{:keys [id value options validation] :as opts}]
+  (let [language @(rf/subscribe [:language])]
```

Comment:
I think it would be nicer if you'd push this subscription one level higher so that `option-field` would be a pure functional component and the user would pass the language. It would make it a little better to show in the examples then too.